dev/CommonStyles/Button_themeresources.xaml

```diff
@@ -62,16 +62,16 @@
             <StaticResource x:Key="AccentButtonBorderBrushPressed" ResourceKey="SystemControlHighlightTransparentBrush" />
             <StaticResource x:Key="AccentButtonBorderBrushDisabled" ResourceKey="SystemControlDisabledTransparentBrush" />
             <StaticResource x:Key="ButtonBackground" ResourceKey="SystemControlBackgroundBaseLowBrush" />
-            <StaticResource x:Key="ButtonBackgroundPointerOver" ResourceKey="SystemControlBackgroundBaseLowBrush" />
-            <StaticResource x:Key="ButtonBackgroundPressed" ResourceKey="SystemControlBackgroundBaseMediumLowBrush" />
+            <StaticResource x:Key="ButtonBackgroundPointerOver" ResourceKey="SystemColorHighlightTextColor" />
```

Comment:
@ranjeshj @jevansaks I think I've asked this before, but to double-check: is it ok to use this kind of lookup from a SolidColorBrush to a Color directly?

Reply:
Good catch. I think it works in most cases that it's ok but I prefer to be explicit if we can. I feel like there is a corner case that I never remember where it causes us trouble.

Reply:
So do you want me to change all of the references to brushes then?

Reply:
Sounds like changing them to brushes would be safer. 
Use these: https://github.com/microsoft/microsoft-ui-xaml/blob/c502a3e4af53edde7a0b5f474dc012d8c67fe23c/dev/CommonStyles/Common_themeresources_any.xaml#L573

Reply:
@YuliKl Should we update the references to these colors holistically then just to be sure?

Reply:
Yes, all of these should be point to the brush resource so a SolidColorBrush is not created each time which will affect perf.

`<StaticResource x:Key="ButtonBackgroundPointerOver" ResourceKey="SystemColorHighlightTextColorBrush" />`

These are newly introduced brush resources to WinUI 2.6. They have the same names (with "Brush" added to the end) as the HC API colors making it much easier to understand:

SystemColorWindowTextColorBrush
SystemColorWindowColorBrush
SystemColorButtonFaceColorBrush
SystemColorButtonTextColorBrush
SystemColorHighlightColorBrush
SystemColorHighlightTextColorBrush
SystemColorHotlightColorBrush
SystemColorGrayTextColorBrush

Reply:
Fixed now, @marksfoster @ranjeshj should I also update ALL other references?

Reply:
@chingucoding you mean update any other HC resources that point to the color instead of the brush?

Reply:
Yes, exactly @marksfoster .

Reply:
I think check this PR in and I'll talk to Yulia about this extra refactor work and whether we want to take it on.